examples/configs/pose_graph/pose_graph_synthetic.yaml

```diff
@@ -4,8 +4,8 @@ profile: True
 savemat: True
 
 num_poses: 256
-rotation_noise: 0.05
-translation_noise: 0.02
+rotation_noise: 0.02
+translation_noise: 0.05
```

Comment:
```
translation_noise: 0.05
rotation_noise: 0.02
```
so we follow consistent convention for this example: translation then rotation.

Reply:
Fixed.